fs/dafsserver/dafsserver.cpp

```diff
@@ -5611,7 +5622,8 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                     }
                     if (exception)
                     {
-                        EXCLOG(exception, "CRemoteFileServer (row service)");
+                        if (exception->errorCode() != DAFSERR_serveraccept_fail_portcheck)
+                            EXCLOG(exception, "CRemoteFileServer (row service)");
```

Comment:
see above comment

Reply:
ok, yes, code added.

Reply:
Altho without the port forwarding/mapping of 443 to 7600 I doubt there will be any extraneous connections to the row service port 7601.